### PR TITLE
[BUG] Old screenshot is displaying for Home page

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #908.

Preview [here](https://security-docs_923.docs-preview.app.elstc.co/guide/en/security/master/ingest-data.html#_install_beats_using_the_kibana_ui_guide).

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
Screenshot looks good! In the Security docs, I've seen arrows and the rectangle and square shapes used to highlight certain areas of the UI so that might be one thing to verify before finalizing this change.

**Comment by deepikakeshav-qasource** [comment]
Hi @joepeeples ,

Required changes are now present in the shared guide.

image

Hence, approving the PR request.

Thanks!!

**Review by deepikakeshav-qasource — APPROVED** [review]
Approved!!

File: docs/getting-started/ingest-data.asciidoc [hunk 1]
```diff
@@ -55,8 +55,7 @@ You can install {beats} using the {kib} UI guide or directly from the command li
 [discrete]
 === Install {beats} using the {kib} UI guide
 
-Click *Home* -> *Add events*, and follow the links for the types of data you want to
-collect.
+Click *Home* -> *Add data*, and follow the links for the types of data you want to collect.
```
Comment (jmikell821) [inline docs/getting-started/ingest-data.asciidoc:58]:
```suggestion
Click *Home* -> *Add data*, then follow the links for the types of data you want to collect.
```

**Review by jmikell821 — APPROVED** [review]
Super small nit, then OK to merge/backport.

**Comment by joepeeples** [comment]
> Screenshot looks good! In the Security docs, I've seen arrows and the rectangle and square shapes used to highlight certain areas of the UI so that might be one thing to verify before finalizing this change.

@nastasha-solomon Thanks for the heads-up! I looked through various images in `security-docs` and there seems to be a range of styles, though rectangles are much more common so I changed those. I tried adding arrows but it got a bit busy so I left those out.

**Pull request merged**